app/backplane/sensor_module/src/c_sensing_tenant.cpp

```diff
@@ -13,7 +13,7 @@ LOG_MODULE_REGISTER(CSensingTenant);
 
 CSensingTenant::CSensingTenant(const char* name, CMessagePort<NTypes::SensorData>& dataToBroadcast,
                                CMessagePort<NTypes::SensorData>& dataToLog, CDetectionHandler& handler)
-    : CTenant(name), dataToBroadcast(dataToBroadcast), dataToLog(dataToLog), detection_handler(handler),
+    : CTenant(name), dataToBroadcast(dataToBroadcast), dataToLog(dataToLog), detectionHandler(handler),
```

Comment:
im being silenced for my beliefs (my incorrect casing)

Reply:
Do it in C not C++ smh